waku/waku.md

```diff
@@ -92,15 +91,28 @@ rate-limits = "[" limit-ip limit-peerid limit-topic "]"
 
 light-node = BIT
 
+pow-requirement-key = 0
+bloom-filter-key = 1
+light-node-key = 2
+confirmations-enabled-key = 3
+rate-limits-key = 4
+topic-interest-key = 5
+
 status = "[" 
-        version pow-requirement 
-        [ bloom-filter ] [ light-node ] 
-        [ confirmations-enabled ] [ rate-limits ]
+        version
+        [ pow-requirement-key pow-requirement ]
```

Comment:
So here we have `[version, [key1, val1], [key2, val2], ...]`. What if we could do:
```
[version, [[key1, val1], [key2, val2], ...]]
```
and have both `version` and this second list required? That would make the spec more clear. All elements in the inner list are optional.

Reply:
Ack, agree, will change

`[version options]`

`opts  = [[k1 v1] [k2 v2] ...]`